h5netcdf/tests/test_h5netcdf.py

```diff
@@ -2161,7 +2211,7 @@ def test_array_attributes(tmp_local_netcdf):
         np.testing.assert_equal(ds.attrs["empty_list"], np.array([]))
         np.testing.assert_equal(ds.attrs["empty_array"], np.array([]))
 
-    with netCDF4.Dataset(tmp_local_netcdf, mode="r") as ds:
+    with netCDF4.Dataset(tmp_local_netcdf, mode="r", backend=backend) as ds:
```

Comment:
This looks wrong, `netCDF4.Dataset` does not support the pyfive backend :)

Reply:
Indeed it does not :) 2b12af7dda92035011fb1df0bcb8456ecd47f037